ApnsPHP/Push/Server.php

```diff
@@ -239,15 +239,27 @@ public function add(Message $message)
         $n++;
     }
 
+    /**
+     * Legacy alias for getMessageQueue()
+     *
+     * @see getMessageQueue()
+     * @deprecated remove in 3.0
+     */
+    public function getQueue($empty = true)
+    {
+        $this->logger()->info('getQueue() is deprecated, use getMessageQueue() instead.');
```

Comment:
PSR-3 mentions "deprecated APIs" as an example for the "warning" level

Reply:
Fixed